clang/unittests/Format/FormatTestJS.cpp

```diff
@@ -579,6 +579,14 @@ TEST_F(FormatTestJS, GoogScopes) {
                "});");
 }
 
+TEST_F(FormatTestJS, GoogAnonymousClass) {
```

Comment:
Please add a `TokenAnnotatorTest` to test the annotation of the braces instead of (preferred), or in addition to, this test.

Reply:
Thanks! Added one.